src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrValue.cs

```diff
@@ -44,6 +45,7 @@ public class DkmClrValue : DkmDataContainer
             this.Alias = alias;
             this.EvalFlags = evalFlags;
             this.ValueFlags = valueFlags;
+            this.NativeComPointer = isComObject ? (ulong)1 : 0;
```

Comment:
Is the cast to `ulong` really necessary here? What's the type of `NativeComPointer`?